### PR TITLE
Nb #58 sample data grp 3 4

### PR DESCRIPTION
**Pull request by nyxathid**
- Added `toString()` to `WebPage`
- Modified `LanguageExtractor.extractWETPages(...)` accordingly
- Removed obsolete test run `testLanguageExtractor_super_slow`
- New test run `testExtractAllEnglishPages` to test the functionality of `LanguageExtractor.extractWETPages(...)` 
- New test run `createFilteredSampleDataForGroups3_4` to create some sample date for groups 3 & 4 (issue #58 )

### CODE REVIEW

File: src/web-page.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export class WebPage {
 
     public protocol : string;
-    public headers : object;
+    public headers : any;
```
Comment (lgfrbcsgo) [inline src/web-page.ts]: Why change that to `any`? Isn't the structure of the headers an object with a structure like this `{ [ property : string ] : string }`?

Comment (nyxathid) [inline src/web-page.ts]: That was weird. I could not iterate over all properties of `headers` if a for each loop as long as it remained `object` so I changed it to `any`. `object` is not a basic TypeScript type. I guess it has to be `Object`.

File: src/web-page.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export class WebPage {
 
     public protocol : string;
-    public headers : any;
+    public headers : Object;
```
Comment (lgfrbcsgo) [inline src/web-page.ts]: Good, but have you tested something like `{[property : string] : string}` instead of `Object`?

Comment (nyxathid) [inline src/web-page.ts]: :D

**Pull request merged**